src/utils.ts

```diff
@@ -50,3 +51,7 @@ export let to_lsp_position = (pos: IPosition): Position => {
 export let get_range = (ps: IPositionedString): Range => {
   return _get_range_che(ps);
 };
+
+export let get_golang_imports_cmd = (): string => {
```

Comment:
@dgpatelgit using _ is not a common pattern & it will be blocked soon after #161 . Use camel case.
```suggestion
export const getGoLangImportsCmd = (): string => {
```

Reply:
I followed existing naming pattern, however now modified as per your comments. PTAL.